main.go

```diff
@@ -148,6 +151,24 @@ func main() {
 	Systray.Start()
 }
 
+func panicToFile() {
+	if r := recover(); r != nil {
+		fileName := "crashreport_" + time.Now().Format("20060102150405") + ".txt"
+		currDir, err := osext.ExecutableFolder()
+		if err != nil {
+			panic(err)
```

Comment:
I don't know if it's correct to panic inside a `defer` function. Maybe it's better to log the error. Returning the error is not possible.